research/serializers/group_serializer.py

```diff
@@ -0,0 +1,32 @@
+from research.models.group import Group
+from research.models.study import Study
+from rest_framework import serializers
+
+
+class MiniStudySerializer(serializers.ModelSerializer):
```

Comment:
@ZviBaratz can you comment here as for the MiniStudy group serializer?

Reply:
And could this perhaps be achieved via just fetching the study from the model and adding it's fields to the Meta definition instead of needing to nest the REST interface?

Reply:
I'm not sure I understand what you mean. Are you suggesting something like:

    class GroupReadSerializer(serializers.HyperlinkedModelSerializer):
        url = serializers.HyperlinkedIdentityField(view_name="research:group-detail")

        class Meta:
            model = Group
            fields = ("title", "description", "url", "id", "study__title")

Because that raises a `django.core.exceptions.ImproperlyConfigured`. 
I think that according to the [serializer relations API guide](https://www.django-rest-framework.org/api-guide/relations/) this is the standard way to include such information.

Reply:
So , but here - you are using a `HyperLinkedIdentifyField` which allows to set and read the group-detail,  and this is GroupReadSerializer.